src/app/showcase/components/context-menu/showcase-context-menu.component.ts

```diff
@@ -22,13 +24,31 @@ export class ShowcaseContextMenu implements OnInit {
 			new ContextMenuOption('option3', 'Option 3', null, null, false, 'icon-chevron-circle-up', 'rgb(50, 50, 50)', 'rgb(21, 143, 239)')
 		];
 
+		this.contextMenuOptions3 = [
+			new ContextMenuOption('option6', 'Option 6', null),
+			new ContextMenuOption('option7', 'Option 7', null),
+		];
+
+		this.contextMenuOptions4 = [
```

Comment:
Add an example with Scroll. I tried it adding more items and the scroll does not appear.